### PR TITLE
WIP to remove pre compile

### PR DESCRIPTION
**Pull request by ScottDowne**

### CODE REVIEW

**Comment by ScottDowne** [comment]
@alicoding maybe time to start looking at this, if possible.

@alanmoo I might need to lean on you a bit for review, if it goes to next week.

File: package.json [hunk 1]
```diff
@@ -9,8 +9,8 @@
     "build:exchangerates": "node scripts/exchangerates.js",
     "build:babel": "babel src --compact=true -s --out-dir dist",
     "build:webpack": "webpack --optimize-minimize --optimize-dedupe",
-    "watch:js": "npm-run-all --parallel watch:webpack watch:babel",
-    "watch:babel": "npm run build:babel -- -w -q",
+    "watch:js": "npm-run-all --parallel watch:babel watch:webpack",
```
Comment (alicoding) [inline package.json]: does the sequence in this matter?

File: tests/http/bad-actors.test.js [hunk 2]
```diff
@@ -1,16 +0,0 @@
-var server = require('../../server');
```
Comment (alicoding) [inline tests/http/bad-actors.test.js]: why this test was removed?

**Comment by alicoding** [comment]
Just one redflag about that test you've removed since that was a bug that we may hit because of how hapijs handle their routing and stuff...

File: package.json (hunk 1 above)
Comment (ScottDowne) [inline package.json]: Not really, but webpack listens for changes in babel, and webpack does an initial build, along with babel. So it just felt logical to put them in their logical order.

Also with this order, it more likely to reduce log outputs that and not needed, but that's only on average.

File: tests/http/bad-actors.test.js (hunk 2 above)
Comment (ScottDowne) [inline tests/http/bad-actors.test.js]: I talked to Jbuck, and he created this test to test that urls with non ascii characters sent us to a 400. However, we now redirect to the home for urls that are not in react-router, so I saw no more need to test that we were doing something we are not.

I couldn't figure out what else this test was trying to do beyond that, and there wasn't anything else in the test saying why a url couldn't contain non ascii characters, so I actually consider it a pretty toxic test.

Comment (alicoding) [inline tests/http/bad-actors.test.js]: Please do a `curl` test locally to see if it redirects properly and if so then this is r+

Comment (ScottDowne) [inline tests/http/bad-actors.test.js]: I'm happy to re add this if you can give more info about the bug. Right now, there isn't anything in the test about why it's there.

Although the test would need to be re made, as it's not working like it used to (and hasn't for a while)

**Comment by ScottDowne** [comment]
This is going to staging now.

**Pull request merged**